packages/components/src/DatePicker/DatePicker.tsx

```diff
@@ -83,7 +83,13 @@ interface DatePickerInlineProps extends BaseDatePickerProps {
 
 type DatePickerProps = XOR<DatePickerModalProps, DatePickerInlineProps>;
 
-/*eslint max-statements: ["error", 13]*/
+interface DatePickerCloseDetails {
+  id: string;
+}
+
+export const DATEPICKER_SWITCH_EVENT = "atlantis.datepicker-switch";
```

Comment:
The[index.ts ](https://github.com/GetJobber/atlantis/blob/JOB-116104/JOB-116904/one-datepicker-at-a-time/packages/components/src/DatePicker/index.ts)file will also need to be updated for consumers to use this without importing from the `/dist/` folder

Reply:
thank youuuu, done here 013701bc28fb35f50a808e4c57b50b885733d481

Reply:
I could see the need when people are writing testing and it would be something they could use if needed

Reply:
You are right, I was worried for people writing tests in a weird way but we should actually expect them to use "standard" ways of testing things

Reply:
reverted exporting the event name here d016f5e74226fb2f429c59968a33d8fdc26a58b6